card-mod-export/set/abstract/export.rb

```diff
@@ -59,7 +65,7 @@ def export_ok?
   end
 
   def export_formats
-    %i[csv json]
+    %i[csv json jsonld]
```

Comment:
This should probably not be here, because it will add this regardless of whether jsonld is supported. I think it would be better in that abstract module (eg `Abstract::JsonldSupport`) that I mentioned elsewhere.